app/[lang]/pages/dashboard/page.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
```

Comment:
I think the `pages`-folder is unnecessary. I think it's simpler if `/dashboard` leads to the dashboard-page instead of `/pages/dashboard`